### DISCUSSION TITLE
Reconsider the Manual Support in GitHub Education

### DISCUSSION THREAD

**Discussion by ianchb** [community/community#63158] [GitHub Education]
### Hi everyone,

I strongly oppose the recent policy change regarding the denial of requests for manual support in GitHub Education. While I understand your decision to shift some support to the GitHub Education community, I believe it is unfair for users who require personal assistance with global campus application reviews and application status updates. I urge you to reconsider this policy and provide appropriate manual support.

In your notification, you mentioned prioritizing the development and improvement of resources such as FAQs and video tutorials to help users find answers and resolve issues. While these efforts aim to provide a better support experience, they do not address all cases where manual support is essential.

You mentioned improving the global campus verification process to provide clearer feedback, but I believe this does not solve the need for additional guidance in understanding the reasons for denial in some cases. Sometimes, users may require communication with a real person to better comprehend the reasons for denial and seek solutions. I hope you can reconsider and provide an appropriate channel for users who need personal support.

While you suggest that users search for similar issues and carefully read documentation and FAQs, this does not always resolve all problems. In certain situations, which may be unique, users need to engage with someone who can understand their specific issues. In such cases, manual support is crucial.

Therefore, I strongly urge you to reconsider this policy and provide an appropriate channel for users who require personal support. This will help ensure that every user receives fair and personalized assistance, rather than solely relying on self-help resources.

Thank you for your attention to my concerns. I hope you will seriously consider my objections and take the necessary steps to improve the support experience in GitHub Education.

Sincerely,
Cheng Haobo

**Comment by humairshoukat** [comment]
Try sharing this feedback with a GitHub Campus Expert. He/She can raise and internal issue on this. Find the nearest one here https://education.github.com/experts

**Comment by hardianlawi** [comment]
Agree with the above.

I have been trying to sign up for the education pack with the exact same steps as my fellow students and still couldn't get approved. Removing manual human review works only if I could understand what am I doing wrong. I keep getting the same messages that I believe I have followed..
